src/org/traccar/protocol/OwnTracksProtocolDecoder.java

```diff
@@ -80,16 +85,29 @@ protected Object decode(
         if (root.containsKey("t")) {
             position.set("t", root.getString("t"));
         }
+        if (root.containsKey("p")) {
+            position.set("pb", root.getInt("p"));
```

Comment:
`pb` was atmospheric pressure; it is reported by our iOS app on iPhone >= 6. I've removed it for the time being; it's not so important.